docusaurus/docs/user-docs/settings/single-sign-on.md

```diff
@@ -16,7 +16,8 @@ To configure the SSO feature settings:
 
 | Setting name      | Instructions      |
 | ----------------- | ---------------------|
-| Auto-registration | Click on **ON** to allow the automatic creation of a new Strapi administrator when an SSO login does not match an existing Strapi administrator account. If this setting is set on **OFF**, new Strapi administrators accounts must be created manually beforehand. |
+| Auto-registration | Click on **True** to allow the automatic creation of a new Strapi administrator when an SSO login does not match an existing Strapi administrator account. If this setting is set on **False**, new Strapi administrators accounts must be created manually beforehand. |
 | Default role      | Choose among the drop-down list the role to attribute by default to auto-registered Strapi administrators through SSO login.           |
+| Local authentication lock-out | Choose among the drop-down list the [roles](/user-docs/users-roles-permissions) for which the local authentication capabilities are disabled.<br />Users locked out of local authentication will be forced to use SSO to login and will not be able to change or reset their password. |
```

Comment:
If "super admin" is selected in the roles list, it becomes possible to accidentally lock oneself out of the admin entirely (it happened to me during testing).  For local auth we provide `admin:reset-password` as a safeguard, but in this case it won't work because we've disabled local authorizations.

So we may want to add a warning that if "Super Admin" is selected and cannot log in, the only way to get in will be to temporarily disable the SSO feature entirely and log in with username/password to change the settings (like removing the "super admin" role from this list) and then re-enable SSO.